keyboards/ealdin/readme.md

```diff
@@ -0,0 +1,15 @@
+# Quadrant
+
+![Quadrant](https://i.imgur.com/S5f1Tn7.jpg)
+
+A 5x14 ortholinear keyboard with "barebone" design using Elite-C.
+
+Keyboard Maintainer: [Ealdin](https://github.com/Ealdin)  
+Hardware Supported: Quadrant E-C   
+Hardware Availability: Group Buy via /r/mk
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Please add instructions on how to enter the bootloader

Reply:
Updated with a similar estructure to the Exclusive keyboards, seeing as @Ealdin is developing new boards

Reply:
So the bootloader information is at /ealdin/quadrant/readme.md